pipelines/base/role.yaml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
@goern noted that this might be generated, using kubebuilder annotations in `controllers/cnbi/controller.go`. Is there some problems doing it that way ?

Reply:
please see https://github.com/thoth-station/meteor-operator/pull/135/files
maybe kubebuilder is not regenerating the RBAC config?!

Reply:
As the name of roles are currently different 
maybe we can have them for this pr.
and we can remove them in later prs.